src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -71,4 +72,12 @@
     }
     return Collections.emptyList();
   }
+
+  /**
+   * @return {@code [--name, key1=val1, --name, key2=val2, ...]} or {@code []} if
```

Comment:
yeah. unfortunately we are doing this everywhere else in this class too. I don't want to fix those and leak stuff into this PR, but I guess I could kick off using the right format here (instead of continuing the incorrect approach).